tests/unit/test-util/test.artifacts.js

```diff
@@ -55,4 +55,59 @@ describe('prepareArtifactsDir', () => {
     }
   ));
 
+  it('throws error when checking the directory if lacks writing permissions',
+    () => withTempDir(
+    (tmpDir) => {
+      const tmpPath = path.join(tmpDir.path(), 'build');
+      return fs.mkdir(tmpPath, '0622').then(() => {
```

Comment:
@saintsebastian like we discussed previously, the tests that are creating the directory with the custom permissions are not going to work on Windows (and I tried directly, by rebasing temporarily this PR on #766, and it looks that it fails because the custom permissions have no effect on the Windows system: https://ci.appveyor.com/project/rpl/web-ext/build/1.0.15/job/ro3024btux5tnm01#L1513)

Do you mind to:
- open a new issue to "investigate and re-enable this tests on windows"  
- prepare these tests to be skipped (using something like `if (process.platform == "win32") { this.skip(); return }`, 
- add an inline comment near `this.skip()` which reference the newly created issue

NOTE: to be able to use `this.skip()`, the test case needs to be in a regular function (`function () { ... }`) instead of an arrow function (`() => { ... }`), because the arrow function rebind `this` and so it will not provide the `skip` method.